src/dpsvalidationservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/ValidationControllerTest.java

```diff
@@ -38,6 +38,10 @@ public void SetUp() {
         try {
             Mockito.when(dfcrmsApiMock.caseSequenceNumberGet("1234567", "PEL"))
                     .thenReturn(caseSequenceNumberResponse);
+
+            // emulating non 200 response
```

Comment:
do we need a comment on line 39 as well?

Reply:
out of scope for the diff, but was just wondering

Reply:
no I could remove this one, it's just for the dev to show an example of beharivor

Reply:
all good!